src/app/public/modules/phone-field/phone-field.component.spec.ts

```diff
@@ -1589,4 +1611,58 @@ describe('Phone Field Component', () => {
 
   });
 
+  describe('inside input box', () => {
+    let fixture: ComponentFixture<PhoneFieldInputBoxTestComponent>;
+    let nativeElement: HTMLElement;
+    let mockThemeSvc: any;
+
+    beforeEach(() => {
+      mockThemeSvc = {
+        settingsChange: new BehaviorSubject<SkyThemeSettingsChange>(
+          {
+            currentSettings: new SkyThemeSettings(
+              SkyTheme.presets.default,
+              SkyThemeMode.presets.light
+            ),
+            previousSettings: undefined
+          }
+        )
+      };
+
+      TestBed.configureTestingModule({
+        declarations: [
+          PhoneFieldInputBoxTestComponent
+        ],
+        imports: [
+          SkyInputBoxModule,
+          SkyPhoneFieldModule,
+          NoopAnimationsModule,
+          FormsModule
+        ],
+        providers: [
+          {
+            provide: SkyThemeService,
+            useValue: mockThemeSvc
+          }
+        ]
+      });
+
+      fixture = TestBed.createComponent(PhoneFieldInputBoxTestComponent);
+      nativeElement = fixture.nativeElement as HTMLElement;
+    });
+
+    it('should render in the expected input box containers', fakeAsync(() => {
+      detectChangesAndTick(fixture);
+
+      const inputBoxEl = nativeElement.querySelector('sky-input-box');
+
+      const inputGroupEl = inputBoxEl.querySelector('.sky-form-group > .sky-input-group');
+      const countryBtnEl = inputGroupEl.children.item(0);
+      const containerEl = inputGroupEl.children.item(1).children.item(0);
```

Comment:
The relationship between elements is important in knowing that elements get put in the correct spots in the DOM, which is why I'm using things like `children` instead of `querySelector`. Ideally there would be a test fixture for input box, but I was in a chicken/egg situation where I refactored a lot of the input box DOM structure but needed that released before I could write a fixture for it. Maybe some day...